extra-optenvw64/binutils+w64/spec

```diff
@@ -1,3 +1,3 @@
-VER=2.29.1
-SRCTBL="https://ftp.gnu.org/gnu/binutils/binutils-$VER.tar.gz"
-CHKSUM="sha256::0d9d2bbf71e17903f26a676e7fba7c200e581c84b8f2f43e72d875d0e638771c"
+VER=2.34
+GITSRC="https://sourceware.org/git/binutils-gdb.git"
```

Comment:
Please use the new ACBS [spec format](https://wiki.aosc.io/developer/packaging/acbs/spec-format/).